### ISSUE TITLE
Excluded path not honoured

### ISSUE THREAD

**Issue by princeofnaxos** [vslavik/poedit#127]
First, I _must_ say that I find the UI of the Catalog Properties very much undeveloped. One should be able to browse to the directory to use.

But my actual problem is that whatever I add to 'Excluded paths' has no effect. I try to exclude one folder that is contained within the included paths (which is '.' [dot]). Then I added './js' to the Excluded section, but when updating, I still get texts from there.

**Comment by numediaweb** [comment]
I confirm this bug still exist, my folder structure is this:
```
project
│   README.md
└───languages
    │   my.pot
└───includes
    ├───lib
    │   │   ├───Twig
    │   │   ├───ezSQL
```

And my pot file settings is like:

![bug_view_01](https://cloud.githubusercontent.com/assets/288614/8278228/21f121e8-18ca-11e5-81f6-a6e08d158e34.png)

Note that I excluded the lib folder, but still I get this error:

![bug_view_02](https://cloud.githubusercontent.com/assets/288614/8278230/24f24bce-18ca-11e5-9950-48e34956c102.png)

Normally it shouldn't look inside the lib folder right?

Thanks :)

**Comment by vslavik** [comment]
> I confirm this bug still exist in version 1.8.1,

Nope, completely unrelated. See above, *this* issue is about incorrectly entered path. Your very own screenshots show that this class of error is *impossible* in 1.8's UI.

> my folder structure is this:

Pretty, but it would be much more useful to get the actual folder, so that I have a chance to reproduce it. In absence of that, I'm going to assume that it was introduced very recently and fixed in 027fe0fb. Should be OK in 1.8.2 (soon), please let me know if it won't.
